autokeras/tasks/image.py

```diff
@@ -67,22 +72,22 @@ def __init__(self,
             seed=seed)
 
     def fit(self,
-            x=None,
-            y=None,
-            epochs=None,
-            callbacks=None,
-            validation_split=0.2,
-            validation_data=None,
+            x: Union[np.ndarray, tf.data.Dataset] = None,
+            y: Union[np.ndarray, tf.data.Dataset] = None,
+            epochs: Optional[int] = None,
+            callbacks: Optional[List[tf.keras.callbacks.Callback]] = None,
+            validation_split: Optional[float] = 0.2,
+            validation_data: Any = None,
```

Comment:
Change to `validation_data: Optional[Union[np.ndarray, tf.data.Dataset]] = None,`